src/server/api/controllers/orders.controller.js

```diff
@@ -0,0 +1,44 @@
+const knex = require('../../config/db');
+const HttpError = require('../lib/utils/http-error');
+
+  
+const getOrderById = async(id) => {
+    if (!id) {
```

Comment:
Note that in HTTP, all parameters are strings. 
Currently this if statement assumes the "larger than 0" part, but the string "0" is a truthy value.

I'd recommend using `parseInt` to make sure you're working with numbers, like this:
```
const user_id = parseInt(query_user_id);
if (isNaN(user_id) || user_id < 1) {
 ...
```